src/app/find-help/accommodation-search/accommodation-search.component.ts

```diff
@@ -68,6 +68,9 @@ export class AccommodationSearchComponent implements OnInit {
         this.results = results.content ?? [];
         this.total = results.totalElements;
         this.loading = false;
+        setTimeout(() => {
+          document.getElementById('results_start')?.scrollIntoView();
```

Comment:
Don't use document traversal, set a ref to the element instead. See https://www.pluralsight.com/guides/how-to-use-template-reference-variables-in-angular, section "Using Template Reference Variables with Child Components"

Reply:
Thanks for the tip, done.